docs/troubleshoot/troubleshoot-apiml-error-codes.md

```diff
@@ -629,7 +665,7 @@ The following error message codes may appear on logs or API responses. Use the f
 
   **Reason:**
 
-  The Zaas Client configuration does not contain the path to the trust store.
+  The Zaas Client configuration didn't contain the path to the trust store
```

Comment:
Keep verb in the present tense and avoid contractions. "The Zaas Client configuration does not contain the path to the trust store" is correct.